src/block/BlockComponents.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace customiesdevs\customies\block;
+
+use customiesdevs\customies\block\component\BlockComponent;
+
+interface BlockComponents {
+
+    /**
+     * Add component adds a component to the item that can be returned in the getComponents() method to be sent over
```

Comment:
block not item

Reply:
that text was a copy and paste from the Item Component file